### PR TITLE
Add non-functioning items to the nav drawer.

### PR DESCRIPTION
**Pull request by JakeWharton**
device-2015-03-27-014242

### CODE REVIEW

File: src/main/res/layout/main_drawer_item.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:textSize="16sp"
+    android:textColor="?android:attr/textColorPrimary"
+    android:paddingTop="12dp"
+    android:paddingBottom="12dp"
+    android:paddingLeft="16dp"
+    android:paddingRight="16dp"
+    android:drawablePadding="16dp"
+    android:background="?android:attr/selectableItemBackground"
```
Comment (JakeWharton) [inline src/main/res/layout/main_drawer_item.xml]: Could be a style... but since the layout is the component of re-use I didn't bother.

**Comment by JakeWharton** [comment]
Going to update this to material keylines before merge.

**Comment by JakeWharton** [comment]
Well that was easy...

**Comment by mattprecious** [comment]
lgtm!

**Pull request merged**